<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWindowWidget</class>
 <widget class="QMainWindow" name="MenuWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>850</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>850</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #000000
</string>
   </property>
   <widget class="QPushButton" name="NewGameButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>420</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/Images/Images/StartButton.png); border-style: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ContinueButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>420</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/Images/Images/ContinueButton.png); border-style: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="EggsButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>420</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/Images/Images/EggsButton.png); border-style: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="CreditsButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>610</y>
      <width>420</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/Images/Images/CreditsButton.png); border-style: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>730</y>
      <width>420</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/Images/Images/ExitButton.png); border-style: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>521</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: url(:/Images/Images/GameTittleLabel.png); border: 0;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
